uk.co.compucorp.civicrm.tasksassignments/js/controllers/assignments.js

```diff
@@ -0,0 +1,8 @@
+define(['controllers/controllers', 'services/contact'],
+    function (controllers) {
+        controllers.controller('AssignmentsCtrl', ['$scope', '$log', '$modal', '$rootElement', '$rootScope', '$state', 'config',
+
+            function ($scope, $log, $modal, $rootElement, $rootScope, $state, config) {
+                $log.info('Controller: AssignmentsCtrl');
+            }]);
+    });
```

Comment:
Can you make sure to set up your editor so it automatically leaves, on save, a new line at the end of the file?